doc/integrator/filter.rst

```diff
@@ -118,13 +118,22 @@ Simple example:
 
 Step 2: Administration interface
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+As a final step, if you wish to override the default behavior regarding how the GeoMapFish filter will provide filter attributes to the user,
+you can define this in the administration interface as follows.
```

Comment:
Cat it be reword like:

You can add some additional configuration in the administration interface as follows.